e2e/ui/global-setup.js

```diff
@@ -0,0 +1,24 @@
+const { chromium } = require('@playwright/test');
+
+module.exports = async config => {
+
+  var NOMAD_TOKEN = process.env.NOMAD_TOKEN;
+  if (NOMAD_TOKEN == undefined || NOMAD_TOKEN == "") {
```

Comment:
```suggestion
  if (NOMAD_TOKEN === undefined || NOMAD_TOKEN === "") {
```

JavaScript likes things to be super equal 😄